web/assets/main.js

```diff
@@ -14,31 +14,39 @@ if (header) {
     new CrossServiceHeader(header).init();
 }
 
+const backLink = document.querySelector('.govuk-back-link');
+if (backLink) {
+    backLink.addEventListener('click', function (e) {
+        window.history.back();
+        e.preventDefault();
+    }, false);
+}
+
```

Comment:
Cool, I hadn't seen this functionality in the past. SAves us from having to roll our own URL saving.